### PR TITLE
docs: code triage

### LINKED ISSUE #81: Code checks and deployment automation
We should consider code deployment and checks automation:
  - [ ] consider using [Code Triage](https://www.codetriage.com/) to attract devs, help solve issues;
  - [x] consider using [Danger](https://github.com/danger/danger); probably already using [ESlint](https://eslint.org/);
  - [x] consider using [mention-bot](https://github.com/facebookarchive/mention-bot);
  - [x] consider using [Greenkeeper](https://greenkeeper.io/).

### PR DESCRIPTION
**Pull request by TobiTenno**
closes #81

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,10 @@
   <a href="https://www.npmjs.com/package/romcal/v/dev" target="_blank" rel="noopener noreferrer"><img alt="dev" src="https://img.shields.io/npm/v/romcal/dev?style=flat-square&logo=npm"></a>
 </p>
 
+<p align="center">
+  <a href="https://www.codetriage.com/romcal/romcal" target="_blank" rel="noopener noreferrer"><img alt="Code Triage Hellpers" src="https://www.codetriage.com/romcal/romcal/badges/users.svg" /></a>
```
Comment (tukusejssirs) [inline README.md]: Could we use shields.io for badges please? Also there is a typo (`Hellpers`).

**Review by tukusejssirs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TobiTenno) [reply README.md]: whoopsies on the typo. i'll check about shields. i'm not seeing any support for it.

Comment (TobiTenno) [reply README.md]:
```suggestion
  <a href="https://www.codetriage.com/romcal/romcal" target="_blank" rel="noopener noreferrer"><img alt="Code Triage Helpers" src="https://www.codetriage.com/romcal/romcal/badges/users.svg" /></a>
```

Comment (TobiTenno) [reply README.md]: @tukusejssirs let me know if you'd like me to keep looking or do something else

Comment (tukusejssirs) [reply README.md]: I think it won’t be possible to use shields.io, as we cannot get the codetriage users number (I could not find an API or something we could use) … unless we simply remove it.

Therefore, I think this is good to go. :pray:

**Review by tukusejssirs — APPROVED** [review]

**Pull request merged**